Games/types/Mafia/roles/cards/JailTarget.js

```diff
@@ -24,9 +25,10 @@ module.exports = class JailTarget extends Card {
             "Jail": {
                 actionName: "Execute Prisoner",
                 states: ["Night"],
-                flags: ["group", "speech", "voting", "anonymous"],
+                flags: ["exclusive", "group", "speech", "voting", "anonymous"],
```

Comment:
even if we do not specify the flag here, the meeting is "effectively exclusive" for the jailor because meeting.exclusive is true